src/main/kotlin/no/nav/familie/ef/sak/vedtak/dto/VedtakDto.kt

```diff
@@ -231,13 +229,19 @@ fun Vedtak.tilVedtakDto(): VedtakDto =
                 )
             }
         }
-        ResultatType.SANKSJONERE -> Sanksjonert(
-            sanksjonsårsak = this.sanksjonsårsak ?: error("Sanksjon mangler årsak."),
-            periode = this.perioder?.perioder?.single()?.fraDomeneForSanksjon()
-                ?: this.barnetilsyn?.perioder?.single()?.fraDomeneForSanksjon()
-                ?: error("Mangler perioder for sanksjon"),
-            internBegrunnelse = this.internBegrunnelse ?: error("Sanksjon mangler intern begrunnelse.")
-        )
+        ResultatType.SANKSJONERE -> {
+            val vedtaksperiodeOvergangsstønad = this.perioder?.perioder?.single()
```

Comment:
Litt tungt for hode å "lese" dette, men går kanskje greit? Vurdere en 
```
when {
this.perioder -> 
this.barnetilsyn ->
}
``` 
hvis det kan funke?

Reply:
Jeg er enig, har en tanke om å skrive om disse periodene med sealed interface, så blir det litt enklere :) Men gjør det i en egen PR